src/warnet/warnet.py

```diff
@@ -187,10 +187,16 @@ def apply_zone_file(self):
     def connect_edges(self):
         for edge in self.graph.edges():
             (src, dst) = edge
-            src_tank = self.tanks[int(src)]
+            src_tank = self.tanks[src]
             dst_ip = self.tanks[dst].ipv4
-            logger.info(f"Using `addpeeraddress` to connect tanks {src} to {dst}")
-            cmd = f"bitcoin-cli addpeeraddress {dst_ip} 18444"
+            # <= 20.2 doesn't have addpeeraddress
+            res = version_cmp_ge(src_tank.version, "0.21.0")
```

Comment:
could this be replaced with a simpler check? just like, 
try:
  addpeeraddress
except:
  addnode
  log